misc-utils/uuidd.c

```diff
@@ -638,52 +632,78 @@ int main(int argc, char **argv)
 		}
 	}
 
-	if (strlen(socket_path) >= sizeof(((struct sockaddr_un *)0)->sun_path))
-		errx(EXIT_FAILURE, _("socket name too long: %s"), socket_path);
+}
 
-	if (!no_pid && !pidfile_path)
-		pidfile_path = UUIDD_PIDFILE_PATH;
+int main(int argc, char **argv)
+{
+	const char	*err_context = NULL;
+	char		*cp;
+	int		ret;
+
+	struct uuidd_cxt_t uuidd_cxt = { .timeout = 0 };
+	struct uuidd_options_t uuidd_opts = { .socket_path = UUIDD_SOCKET_PATH };
```

Comment:
Maybe we can use `cxt` and `opts`, it means without uuidd_ prefix. The result will be a shorter code as it's used everywhere in main().